io.openems.edge.application/EdgeApp.bndrun

```diff
@@ -1,22 +1,19 @@
--runfw: org.eclipse.osgi
+Bundle-Version:				1.0.0.${tstamp}
+Bundle-SymbolicName:		io.openems.edge.application.launch
+JPM-Command:				openems-edge
 
--runprovidedcapabilities: ${native_capability}
-
--resolve.effective: active
+-runfw: \
+	org.eclipse.osgi
 
 -runproperties: \
-	org.osgi.service.http.port=8080,\
-	felix.cm.dir=c:/openems-config
-
--runee: JavaSE-1.8
+	configFile=./resources/config.json,\
+	felix.cm.dir=c:/openems-config,\
+	org.ops4j.pax.logging.DefaultServiceLog.level=INFO,\
+	org.ops4j.pax.logging.service.frameworkEventsLogLevel="DISABLED"
```

Comment:
Did you do those changes to EdgeApp.bndrun? Please make sure to revert as far as possible back to the EdgeApp.bndrun from develop branch.

This mainly applies to all other files as well, that you did not explicitely touch.